junit/src/main/java/org/kaazing/k3po/junit/rules/Latch.java

```diff
@@ -93,7 +93,7 @@ void notifyFinished() {
         switch (state) {
         case INIT:
             notifyPrepared();
-            break;
+//            break;
```

Comment:
okay, perhaps renaming the latched would make it more clear to me.  Can we remove the commented out code (rather then comment it out)